docs/content/en/docs/Tutorials/install-k3os/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "How to install k3OS"
```

Comment:
Great! You can link it from the "0. Understanding the technologies"/"Installing operating systems from a USB-stick" chapter.